mmv1/templates/terraform/examples/dataplex_task_primary.tf.erb

```diff
@@ -0,0 +1,31 @@
+data "google_project" "project" {
```

Comment:
This test should be renamed from "primary" to "basic" for consistency with other samples.